stratum/hal/lib/dummy/dummy_switch.cc

```diff
@@ -290,6 +290,7 @@ ::util::Status DummySwitch::RetrieveValue(
         resp.mutable_sdn_port_id()->set_port_id(
             request.sdn_port_id().port_id());
         status_or_resp = resp;
+	  break;
```

Comment:
tab instead of whitespace

Reply:
We _really_ should implement Stratum wide copts:

```
stratum/hal/lib/dummy/dummy_switch.cc:294:7: warning: unannotated fall-through between switch labels [-Wimplicit-fallthrough]
      default:
      ^
stratum/hal/lib/dummy/dummy_switch.cc:294:7: note: insert 'ABSL_FALLTHROUGH_INTENDED;' to silence this warning
      default:
      ^
      ABSL_FALLTHROUGH_INTENDED;
stratum/hal/lib/dummy/dummy_switch.cc:294:7: note: insert 'break;' to avoid fall-through
      default:
      ^
      break;
```